examples/main.rs

```diff
@@ -1,7 +1,50 @@
-use tiled::Map;
+use std::path::PathBuf;
+
+use tiled::{FilesystemResourceCache, Map};
 
 fn main() {
-    let map = Map::parse_file("assets/tiled_base64_zlib.tmx").unwrap();
-    println!("{:?}", map);
-    println!("{:?}", map.tileset_by_gid(22));
```

Comment:
I messed up, right? Why are things like these counted as changes in respect to master?